packages/ag-grid-theme/css/uitk-icons.css

```diff
@@ -0,0 +1,229 @@
+@font-face {
+  font-family: "uitk-icons";
+  src: url(../fonts/jpmuitk-icons.woff) format("woff");
+}
+
+[class*="ag-theme-uitk"] .ag-icon-aggregation:before {
+  font-family: "uitk-icons";
+  content: "\e974";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-arrows:before {
+  font-family: "uitk-icons";
+  content: "\e970";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-asc:before {
+  font-family: "uitk-icons";
+  content: "\e940";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-cancel:before {
+  font-family: "uitk-icons";
+  content: "\e90e";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-chart:before {
+  font-family: "uitk-icons";
+  content: "\e904";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-columns:before {
+  font-family: "uitk-icons";
+  content: "\e910";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-contracted:before {
+  font-family: "uitk-icons";
+  content: "\e91b";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-copy:before {
+  font-family: "uitk-icons";
+  content: "\e90d";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-cross:before {
+  font-family: "uitk-icons";
+  content: "\e90e";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-cut:before {
+  font-family: "uitk-icons";
+  content: "\e96d";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-csv:before {
+  font-family: "uitk-icons";
+  content: "\e915";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-desc:before {
+  font-family: "uitk-icons";
+  content: "\e93f";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-excel:before {
+  font-family: "uitk-icons";
+  content: "\e91a";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-expanded:before {
+  font-family: "uitk-icons";
+  content: "\e950";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-eye-slash:before {
+  font-family: "uitk-icons";
+  content: "\e966";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-eye:before {
+  font-family: "uitk-icons";
+  content: "\e965";
+}
+
+[class*="ag-theme-uitk"] .ag-icon-filter:before {
+  font-family: "uitk-icons";
+  content: "\e922";
+}
+
+/* [class*="ag-theme-uitk"] .ag-icon-filter-clear:before {
```

Comment:
Missing?

Reply:
yes. Also waiting for design to provide a grab icon (for the drag & drop rows)